### PR TITLE
Add onboarding routes for goals and talking points

### PR DESCRIPTION
**Pull request by manishsaha**
## Overview

Adding last two routes that frontend will need to complete onboarding for users. Adding two new entities - goals and talking points, which allow users to further customize their profile.

## Changes Made

- Two new entities added
- Two new routers added

## Test Coverage

Postman.

### CODE REVIEW

**Review by alanna-zhou — APPROVED** [review]
tried to look for stuff to comment on but everything looks great 👍

File: src/app.ts [hunk 1]
```diff
@@ -27,7 +37,7 @@ DBConnection()
   })
   .catch((error: any) => console.log(error));
 
-async function importDataFromFile(filename: string, fn: (data: string) => Promise<void>) {
+async function importDataFromFile(filename: string, fns: ((data: string) => Promise<void>)[]) {
```
Comment (chalo2000) [inline src/app.ts:40]: what is fns?

File: src/common/constants.ts [hunk 2]
```diff
@@ -29,8 +29,27 @@ const VALID_TIMES = [
...
+  MEETING_SOMEONE_DIFFERENT,
+  LEARNING_FROM_MENTORS,
+  GUIDING_MENTEES,
+];
+
 export default {
   VALID_DAYS,
   VALID_TIMES,
   UNDECLARED_MAJOR,
+  JUST_CHATTING,
```
Comment (chalo2000) [inline src/common/constants.ts]: Not sure if I missed this when looking through the code, but I don't see why we're exporting the individual values (FINDING_MY_PEOPLE, MEETING_SOMEONE_DIFFERENT) and then also exporting a list that contains all those individual values. This seems redundant.

**Review by chalo2000 — APPROVED** [review]
Looks functional to me, just an exporting concern.

File: src/common/constants.ts (hunk 2 above)
Comment (manishsaha) [reply src/common/constants.ts]: ah you are correct, glossed over this part woops

File: src/app.ts (hunk 1 above)
Comment (manishsaha) [reply src/app.ts:40]: I changed it so we can run multiple functions (fns) on the data from one file. This way, we can create a table that is an amalgamation of groups and interests and add it to TalkingPoints without having to iterate over the entire file again.

**Review by leewang-codes — APPROVED** [review]
lgtm!

**Pull request merged**